### ISSUE TITLE
Pressing F1-F4 causes ncpamixer to exit

### ISSUE THREAD

**Issue by mateuszkwiatkowski** [fulhax/ncpamixer#5]
Pressing F1-F4 cuases ncpamixer to exit and it's not possible to change tabs with these keys. I can change tabs using H and L key bindings. The exit code is 0.
I built ncpamixer today using `aur/ncpamixer-git` 

```
$ pacman -Q | egrep "linux 4|ncurses|ncpamixer"
lib32-ncurses 6.0-2
linux 4.7-1
ncpamixer-git c03248b-1
ncurses 6.0-4
$ ncpamixer --version
ncpamixer v1.0
Build type: debug
Build date: 2016-08-17 11:22
```

**Comment by c0r73x** [comment]
This is probably because your terminal sends other keycodes than those specified in the config file. However it shouldn't crash because of it. I will look at it later today.

**Comment by c0r73x** [comment]
I've added a fix in the dev branch. It will be merged in v1.1
